demo/views/demo-all.dust

```diff
@@ -34,14 +35,11 @@
   </div>
 
   <!-- Scripts -->
-  <!-- <script src="../../dist/bluemix-components.min.js"></script> -->
-  <script src="/demo.js"></script>
+  <script src="/carbon-components.min.js"></script>
+  <!-- <script src="/demo.js"></script> -->
```

Comment:
Ah, weird. I'll double check. Also - we decided to remove any javascript from the time picker, and just have it be a text input field and then the user can choose to add the select boxes if he/she needs to.